charts/cactus-backend/templates/configmap.yaml

```
@@ -0,0 +1,13 @@
+{{- if .Values.configFiles.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "cactus-backend.fullname" . }}-config
```

A configmap implies that it's configuration. So I don't think we need the `-config` suffix.

```suggestion
  name: {{ include "cactus-backend.fullname" . }}
```

If we wanna be very specific that each key in the map is a file that will be mounted on the disk, maybe we could do this but I think we can go with the first one instead
```suggestion
  name: {{ include "cactus-backend.fullname" . }}-config-files
```